htdocs/class/xoopsform/formselect.php

```diff
@@ -203,7 +203,11 @@ public function render()
         $ele_title   = $this->getTitle();
         $ele_value   = $this->getValue();
         $ele_options = $this->getOptions();
-        $ret         = '<select size="' . $this->getSize() . '"' . $this->getExtra();
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $ret = '<div class="form-group form-inline"><select class="form-control" size="' . $this->getSize() . '"' . $this->getExtra();
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->